ara_plumes/models.py

```diff
@@ -780,3 +762,61 @@ def background_subtract(
         clean_vid[i] = cv2.subtract(frame, background_img_np)
 
     return clean_vid
+
+
+def flatten_edge_points(mean_points: PlumePoints, vari_points: PlumePoints) -> Float2D:
+    """
+    Convert points on center and edge to distances from each other for a given frame.
+    NOTE: if no pair can be form, points are rejected. If more than one candidate
+    pair is found, the first pair is selected.
+    """
+    pairs = _create_radius_pairs(mean_points, vari_points)
+    rad_dist = []
+    for radius, center_points, edge_points in pairs:
+        dist = np.linalg.norm(center_points - edge_points)
+        rad_dist.append((radius, dist))
+    return np.array(rad_dist)
+
+
+def _create_radius_pairs(
+    mean_points: PlumePoints, vari_points: PlumePoints
+) -> List[tuple[float, Float1D, Float1D]]:
+    """
+    Create pairs of coordinates points based on same radius.
+    NOTE: In the event multiple pairs canddidates are identified, the first
+    pair is selected. If no pairs are identified, point is thrown out.
+    """
+    pairs = []
+    for r, x, y in vari_points:
+        mask, *_ = np.where(np.isclose(mean_points[:, 0], r))
+        if mask.size == 0:
+            continue
+        pairs.append((r, mean_points[mask][0, 1:], np.array([x, y])))
+    return pairs
+
+
+def _sol_in_contour(
+    sols: List[tuple[int, int]], selected_contours: Contour_List
```

Comment:
NIT: Similar to before, I don't know what a `sol` means.  Maybe just `points` with function name `_points_in_contour` (or `pts`